datafusion/core/src/physical_plan/common.rs

```diff
@@ -192,8 +192,10 @@ pub(crate) fn spawn_execution(
 
         while let Some(item) = stream.next().await {
             // If send fails, plan being torn down,
-            // there is no place to send the error
-            output.send(item).await.ok();
+            // there is no place to send the error.
+            if let Err(_) = output.send(item).await {
+                return;
```

Comment:
The change looks reasonable, but I think we should at least log a debug or warning message rather than silently consuming the error.

Reply:
in https://github.com/apache/arrow-datafusion/pull/3350